src/locales/en.json

```diff
@@ -147,6 +147,10 @@
     "repeated": "Household member [{{ percentage }} chance of transmission per week]",
     "partner": "Partner / spouse [{{ percentage }} chance of transmission per week]",
     "indoor": "Indoor",
+    "filtered": "Indoors with a HEPA filter (flow rate 5x room size)",
+    "transit": "An airplane or train with air filtration",
+    "carWindowsDown": "A moving car with the windows rolled down",
```

Comment:
I noticed you used camel casing here. We're using underscores elsewhere.  It'd be good to keep it consistent. I don't care which one we use.

Reply:
Fixed